patch/driver-hwmon-pmbus-dps200.patch

```diff
@@ -0,0 +1,176 @@
+Add PMBUS driver for DPS200 Power Supply
+
+From: schobtr <schobtr@celestica.com>
+
```

Comment:
Please configure your full name, then `git commit -s --amend`, and create the patch file with `git format-patch -1`.

Reply:
The module author is *Pradchaya Phucharoen*?

Reply:
> Please configure your full name, then `git commit -s --amend`, and create the patch file with `git format-patch -1`.

I have updated. Please, review again.

Reply:
> The module author is _Pradchaya Phucharoen_?

Yes, the author of this driver is Pradchaya Phucharoen.